client/src/views/PatchNotesView.tsx

```diff
@@ -32,7 +32,7 @@ const PatchNotesView = () => {
       </Paper>
       {displayedReleases.map((release) => (
         <Paper key={release.version} sx={{ bgcolor: 'background.paperLight', my: 3 }}>
-          <Text h4 bold upperCase>v{release.version} <Text component="span" italic upperCase>{moment(release.date).format('DD MMMM YYYY')}</Text></Text>
+          <Text h4 bold upperCase>v{release.version} <Text component="span" italic upperCase>{dayjs(release.date).format('DD MMMM YYYY')}</Text></Text>
```

Comment:
The original was missing the utc usage

```suggestion
          <Text h4 bold upperCase>v{release.version} <Text component="span" italic upperCase>{dayjs.utc(release.date).format('DD MMMM YYYY')}</Text></Text>
```